client/state/connected-applications/actions.js

```diff
@@ -0,0 +1,58 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import {
+	CONNECTED_APPLICATION_DELETE,
+	CONNECTED_APPLICATION_DELETE_SUCCESS,
+	CONNECTED_APPLICATIONS_RECEIVE,
+	CONNECTED_APPLICATIONS_REQUEST,
+} from 'state/action-types';
+
+/**
+ * Returns an action object to signal the request of the user's connected applications.
+ *
+ * @return {Object} Action object
+ */
+export const requestConnectedApplications = () => ( {
+	type: CONNECTED_APPLICATIONS_REQUEST,
+	meta: {
+		dataLayer: {
+			trackRequest: true,
```

Comment:
Really necessary or can we rely on request de-duplication?

Reply:
Yup, it seems to be necessary to keep the same functionality that the current UI offers, see https://github.com/Automattic/wp-calypso/pull/24175/files#diff-1a0e6fadd9a8706e97f9d77890c2ca38R120